pkg/kubefed2/federate/federate.go

```diff
@@ -138,18 +140,23 @@ func (j *federateResource) Run(cmdOut io.Writer, config util.FedConfig) error {
 	return err
 }
 
-func FederateResource(hostConfig *rest.Config, qualifiedTypeName, qualifiedResourceName ctlutil.QualifiedName, dryrun bool) (*unstructured.Unstructured, error) {
+func FederateResource(hostConfig *rest.Config, qualifiedTypeName, qualifiedName ctlutil.QualifiedName, dryrun bool) (*unstructured.Unstructured, error) {
 	typeConfig, err := lookupTypeDetails(hostConfig, qualifiedTypeName)
 	if err != nil {
 		return nil, err
 	}
 
-	templateResource, err := getTargetResource(hostConfig, typeConfig, qualifiedResourceName)
+	targetResource, err := getTargetResource(hostConfig, typeConfig, qualifiedName)
```

Comment:
`target` seems better compared to `managed` here because we are talking about the resource we want to federate not the resource that federation is managing.

Reply:
I think target is entirely appropriate here.  My comment was that 'target' has been used to describe resources in member clusters and there could be ambiguity if that use continued (rather than switching to the term 'managed' to refer to resources in member clusters).